chapters/ch56.md

```diff
@@ -1,45 +1,45 @@
 
 > Say you have a dataset about customers buying things on a website. A data scientist may have many different ways of analyzing the data. She may draw many different conclusions about whether the website should raise prices, about the lifetime value of customers acquired through different marketing campaigns, and so on. There is no one "right" way to analyze a dataset, and there are many possible useful insights one could draw. Similarly, there is no one "right" way to carry out error analysis. Through these chapters you have learned many of the most common design patterns for drawing useful insights about your ML system, but you should feel free to experiment with other ways of analyzing errors as well.
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử bạn có bộ dữ liệu về khách hàng mua đồ trên một trang mạng. Một nhà khoa học dữ liệu có thể có rất nhiều cách khác nhau để phân tích dữ liệu đó. Người đó có thể đưa ra nhiều kết luận khác nhau như có nên tăng giá, giá trị vòng đời khách hàng đạt được thông qua các chiến dịch tiếp thị khác nhau,... Không chỉ có một cách "đúng" cho việc phân tích dữ liệu, nên có thể có rất nhiều kết luận hữu ích về xu hướng có thể rút ra. Tương tự, không chỉ có một cách "đúng" cho việc thực hiện phân tích lỗi. Thông qua các chương này bạn sẽ học được nhiều mẫu thiết kế phổ biến nhất để rút ra những cái nhìn chính xác nhất về hệ thống học máy của bạn, nhưng bạn cũng nên thử nghiệm các cách phân tích lỗi khác.
 
 > Let’s return to the self-driving application, where a car detection algorithm outputs the location (and perhaps velocity) of the nearby cars, a pedestrian detection algorithm outputs the location of the nearby pedestrians, and these two outputs are finally used to plan a path for the car.
 
---> _replace THIS LINE by your translation for the above line_
+Chúng ta hãy quay trở lại ứng dụng xe tự lái, trong đó thuật toán phát hiện ô tô đưa ra vị trí (và có lẽ là vận tốc) của những chiếc xe gần đó, thuật toán phát hiện người đi bộ đưa ra vị trí của người đi bộ gần đó và hai đầu ra này cuối cùng được sử dụng để hoạch định một con đường cho xe.
 
 ![img](../imgs/C56_01.png)
 
 > To debug this pipeline, rather than rigorously following the procedure you saw in the previous chapter, you could more informally ask:
 
---> _replace THIS LINE by your translation for the above line_
+Để kiểm tra lỗi pipeline này, thay vì tuân thủ nghiêm ngặt quy trình bạn đã thấy trong chương trước, bạn nên đặt những câu hỏi như:
 
 > 1. How far is the Detect cars component from human-level performance at detecting cars?
 
---> _replace THIS LINE by your translation for the above line_
+Cách biệt về chất lượng của việc xác định xe và chất lượng mức con người trong việc xác định xe là bao?
```

Comment:
+ cách biệt về khả năng xác định xe giữa con người và phần thuật toán xác định xe